Airline_Web_API/Airline_Web_API/Helpers/AppException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Airline_Web_API.Helpers
+{
+    public class AppException: Exception
+    {
+        public AppException(): base() { }
+        public AppException(string message): base(message)  {}
+        public AppException(string message, params object[] args)
+             : base(String.Format(CultureInfo.CurrentCulture, message, args))
+        {
+        }
+    }
```

Comment:
What is the purpose of this class? Where is it used?
Also code formatting - separate methods with empty line, put code block on a new line

Reply:
I used it for exception handling but deleted it as it's not suitable for my case